### PR TITLE
[YUNIKORN-319] Simplify the README doc in github repos.

### PR DESCRIPTION
**Pull request by yangwwei**

### CODE REVIEW

**Comment by yangwwei** [comment]
@wilfred-s , @wangdatan  could you pls help to review if you get a chance? Thanks

File: README.md [hunk 1]
```diff
@@ -28,137 +28,33 @@
...
-Currently, we have following channels in the workspace: `#yunikorn-dev` and `#yunikorn-user`.
-
-**Blog posts**
-
-Apache based blogs:
-- [Apache YuniKorn (Incubating) release announcements](https://blogs.apache.org/yunikorn/)
-
-3rd party blog posts:
-- [YuniKorn: a universal resource scheduler](https://blog.cloudera.com/blog/2019/07/yunikorn-a-universal-resource-scheduler/)
+The `yunikorn-core` is the brain of the scheduler, which makes placement decisions (Allocate container X on node Y) according
```
Comment (kingamarton) [inline README.md:59]: Shouldn't we mention here the yunikorn-release repository as well? Mainly because of the helm chart.

Comment (yangwwei) [reply README.md]: Thanks @kingamarton  updated the doc to include the release and the site repo.

**Comment by yangwwei** [comment]
Hi @wilfred-s are we good on this change?

**Pull request closed without merging**